tests/cypress/integration/reportRepair/smellGas.spec.js

```diff
@@ -1,11 +1,33 @@
-describe('smellGas', () => {
+describe('smellGas full version', () => {
```

Comment:
Leaving this as just `smellGas` without the version suffix would be better because once the MVP is no longer required (and as such is deleted), the suffix wouldn't be relevant any longer.

Please delete the version suffix:
```suggestion
describe('smellGas', () => {
```